rust/src/ffi/any.rs

```diff
@@ -354,6 +354,9 @@ pub struct CallbackFn {
 // wrap a CallbackFn in a closure, so that it can be used in transformations and measurements
 pub fn wrap_func(func: CallbackFn) -> impl Fn(&AnyObject) -> Fallible<AnyObject> {
     move |arg: &AnyObject| -> Fallible<AnyObject> {
+        // extends the lifetime of func.callback to the lifetime of this closure
+        let _ = &func.lifeline;
```

Comment:
Is there any particular relationship between this and the similar line in `ffi.rs`? Like, would it serve any purpose to reference the other case in both locations? Perhaps not- If someone needs to touch this again, the connections might be clear to them.

Reply:
They are unrelated, but they do use the same trick!